code/parachain/runtime/common/src/xcmp.rs

```diff
@@ -243,6 +246,8 @@ impl<
 			MultiLocation { parents: 0, interior: X1(GeneralIndex(index)) } =>
 				Some(CurrencyId(index)),
 			_ =>
+			// there is a risk with this `_` that if a new Junction gets added, it will come
```

Comment:
if Root will not configured bad path it will be ok

Reply:
https://composablefinance.slack.com/archives/C031G5NT0CA/p1668168585791579

cc @vimukthi-git

Reply:
not urgent not critical, but can be issue later.